sdcm/nemesis.py

```diff
@@ -818,6 +818,13 @@ def _add_drop_column(self, drop=True, add=True):  # pylint: disable=too-many-bra
                     column_type.forget_variant(added_columns_info['column_types'])
                     del added_columns_info['column_names'][column_name]
 
+    def _add_drop_column_run_in_cycle(self):
+        start_time = time.time()
+        end_time = start_time + 600
+        while time.time() < end_time:
+            self._add_drop_column()
+            time.sleep(1)
+
     def disrupt_add_remove_column(self):
         """
```

Comment:
Is the current c-s version still create standard1 with compact storage?
If so, please send a PR for all longevities to use "pre_create_schema: true" and make sure the pre_create isn't using COMPACT STORAGE